src/main/resources/application.yml

```diff
@@ -10,6 +10,10 @@ spring:
   jpa:
     hibernate:
       ddl-auto: none
+  jackson:
+    deserialization:
+      fail-on-unknown-properties: true
```

Comment:
@stuart-madetech @NyWeb @zackads FYI on this.

This configuration tells Jackson, which is a JSON parsing library for Java and what Spring Boot uses under the hood to parse incoming JSON, that it should fail the request (400) on any fields that it does not recognise.

See [docs](https://docs.spring.io/spring-boot/docs/current/reference/html/appendix-application-properties.html#spring.jackson.deserialization) and [this comment](https://github.com/spring-projects/spring-boot/issues/12684#issuecomment-433803640)

Personally, I think this is worthwhile putting, for now at least, as it ensures that we are not accepting just any fields defined in the JSON payload from a consumer if they are not defined on our model. It should help catch any bugs where we are defining large JSON payloads like we have done for the registration object.